cmd/lambda/credentials_web_page/views/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html> 
+<html>
+    <head> 
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="/api/auth/public/main.css" />
+    <script>
+        var SITE_PATH_PREFIX = "{{.SitePathPrefix}}"
+        var APIGW_DEPLOYMENT_NAME = "{{.ApigwDeploymentName}}"
+        var AWS_CURRENT_REGION = "{{.AwsCurrentRegion}}"
+        var IDENTITY_POOL_ID = "{{.IdentityPoolID}}"
+        var USER_POOL_PROVIDER_NAME = "{{.UserPoolProviderName}}"
+        var USER_POOL_CLIENT_ID = "{{.UserPoolClientID}}"
+        var USER_POOL_APP_WEB_DOMAIN = "{{.UserPoolAppWebDomain}}"
+        var USER_POOL_ID = "{{.UserPoolID}}"
+    </script>
+    </head>
+
+    <body>
+        <script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
+        <script src="/api/auth/public/amazon-cognito-auth.min.js"></script>
+        <script src="https://sdk.amazonaws.com/js/aws-sdk-2.536.0.min.js"></script>
+        <script src="https://unpkg.com/vue"></script>
```

Comment:
Are we actually using vue.js for this?

Reply:
The javascript is structured to use vuejs (i.e. data, methods, and mounted() are all vuejs constructs). I can rewrite it to not use vuejs, but it will take a bit of extra time.

Reply:
yeah, no need to take the time now. 
Just something to keep in mind (probably more dependency bloat than we need for something like this)